app/v1/models/errors/mtdErrors.scala

```diff
@@ -63,11 +63,11 @@ object InvalidStatusError extends MtdError("INVALID_STATUS", "Invalid status", S
 // Rule Errors
 object RuleIncorrectOrEmptyBodyError extends MtdError("RULE_INCORRECT_OR_EMPTY_BODY_SUBMITTED", "An empty or non-matching body was submitted")
 
-object RuleDateRangeInvalidError extends MtdError("INVALID_DATE_RANGE", "Invalid date range, must be 366 days or less", Some(Json.parse(
+object RuleDateRangeInvalidError extends MtdError("INVALID_DATE_RANGE", "Invalid date range, must be 365 days or less", Some(Json.parse(
   """
     |{
     |    "statusCode": 400,
-    |    "message": "INVALID_DATE_RANGE"
+    |    "message": "DATE_RANGE_INVALID"
     |}
     |""".stripMargin)))
```

Comment:
I know these are technically rule errors, but can they move underneath the Legacy errors section - I'd like to move them to a separate error file soon and it'll make it easier if they are all grouped together

Reply:
Sorry think I called it Custom VAT errors